coding_systems/base/tests/dynamic_db_classes.py

```diff
@@ -27,15 +27,13 @@ class DynamicDatabaseTestCase(TestCase):
     """
 
     @property
-    def db_alias(self):
-        """The database alias that needs to be accessed in this test class, str."""
-        raise NotImplementedError(
-            "This test class requires a database alias to be set."
-        )
+    def db_aliases(self):
+        """The database aliases that need to be accessed in this test class, list[str]."""
+        raise NotImplementedError("This test class requires db_aliases to be set.")
```

Comment:
thought: As mentioned in the commit message, I changed this because I thought it was required to solve the problem. It's not, but it allows making the database dependencies more obvious in the updated tests here, so it may be worth retaining.